elvis.config

```diff
@@ -8,7 +8,8 @@
         rules => [{elvis_style, god_modules, #{ignore => [egithub]}},
                   {elvis_style, dont_repeat_yourself, #{min_complexity => 20}},
                   {elvis_style, line_length, #{limit => 120}},
-                  {elvis_style, state_record_and_type, disable}],
+                  {elvis_style, state_record_and_type, disable},
```

Comment:
Whi is this needed? `state_record_and_type` should only apply to OTP behaviors.

Reply:
don't know. wasn't added by me

Reply:
oh sorry it was green you're right!